src/style.css

```diff
@@ -1,3 +1,59 @@
-body {
+html {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: Roboto, serif;
+  font-size: 12px;
+  font-weight: 500;
+  font-style: normal;
+}
+
+.menu {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  height: 60px;
+}
+
+.nav {
+  display: flex;
+  height: fit-content;
```

Comment:
I believe this is not actually the case and height should be `100%`

Reply:
![image](https://github.com/user-attachments/assets/07ef3431-1e7b-4b03-b089-12cb933d8527)
![image](https://github.com/user-attachments/assets/e14037fa-a3ad-4124-b151-ce0098c2d443)
Eu fiz a alteração conforme indicado, mas mesmo assim o resultado continua sendo como não passou em nenhum dos 8 itens avaliado...